src/cascadia/TerminalApp/AppLogic.cpp

```diff
@@ -17,7 +17,7 @@ using namespace winrt::Windows::System;
 using namespace winrt::Microsoft::Terminal;
 using namespace winrt::Microsoft::Terminal::TerminalControl;
 using namespace winrt::Microsoft::Terminal::Settings::Model;
-using namespace ::TerminalApp;
+using namespace ::CommandlineArgs;

```

Comment:
Feels a little gross with it being just in the root namespace and not putting it lower like `::Microsoft::Terminal::CommandlineArgs` or something... but if we want to leave it here since `TerminalApp` was a bare one anyway... I'm not going to complain that hard.

Reply:
To make your life easier, you would need to propagate that change to AppLogic.h, CommandPalette.h, and TerminalPage.cpp